public/src/pages/views/ReleaseView.tsx

```diff
@@ -178,10 +178,9 @@ export class ReleaseView extends React.Component<ReleaseViewProps, ReleaseViewSt
                                 if (current) {
                                     current.released = release;
                                     current.score = scoreFromReviews(current.reviews);
-                                    const ans = this.props.onUpdate(current);
-                                    if (ans) return true;
+                                    const ans = await this.props.onUpdate(current);
+                                    if (ans) return;
```

Comment:
The component calling this function expects a boolean, will it return a boolean?

Reply:
It expects a boolean input (`release`), but a `void` return value. If you hover over the `release` JSX attribute on line 176 (the `release` before the `=` sign), then it will show the `void`. Also, I would expect that this wouldn't compile otherwise. Although, I don't understand why it compiled with the `return true;` statements that were used before...

Reply:
Does it not compile on your end?

Reply:
Oh right, my bad, got lost in the methods. No, everything compiles just fine.